README.md

```diff
@@ -2,23 +2,23 @@
 
 ## What is Olympus?
 
-Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset-backing and protocol owned value.
+Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset-backing and protocol-owned value. Read more about the game theory of Olympus [here](https://olympusdao.medium.com/the-game-theory-of-olympus-e4c5f19a77df).
```

Comment:
- So far we called it PCV and POL. Protocol controlled value and protocol owned liquidity. Below a suggestion to word it properly. 
- I find it to be bad UX to give links names like `here` or `this`. I would like to try keeping link names more expressive. Let me know what you think about this. Below a suggestion for this as well. 

```suggestion
Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset backing and protocol controlled value. Read more about the game theory of Olympus in [one of our blog posts](https://olympusdao.medium.com/the-game-theory-of-olympus-e4c5f19a77df).
```

Reply:
1. Agree on the use of PCV
2. Good suggestion, it does look better and also prepare user for what is to come before clicking the link